android/app/src/main/java/com/expensify/chat/MainActivity.java

```diff
@@ -52,6 +54,11 @@ protected boolean isConcurrentRootEnabled() {
   @Override
   protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(null);
+
+    // Make the app take up the full screen (painted behind transparent status bar)
+    Window w = getWindow();
+    w.setFlags(WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS, WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS);
```

Comment:
No need to do this programmatically, it's less obvious and hard to find IMO. Instead, let's use the style attribute (suggested below).

Reply:
Ah, I take this back. Seems like it is necessary because React Native doesn't use Android layouts.

Reply:
A problem with React native not letting us modify the Android layout is that adding padding to our outermost layout (to solve the overlapping issue) is problematic. I think we'll need to do this: 

- [Get the status bar height](https://stackoverflow.com/a/3356263/4160784), we cannot hardcode this because the size varies by device/screen density
- Programmatically set the View padding to the status bar height, using [something like this](https://developer.android.com/reference/android/view/View.OnApplyWindowInsetsListener#onApplyWindowInsets(android.view.View,%20android.view.WindowInsets))